scripts/diag/tmux-server-contention.py

```
@@ -0,0 +1,198 @@
+    return len(r.stdout.splitlines()) if r.returncode == 0 else 0
+
+
+def arm_up(sock, tag):
+    """Mint a scratch pane running `cat` and tee its output through the stamper.
+
+    `cat` needs no rc file, so nothing in the user's shell config can perturb the
+    timing or leak surprise output into the capture.
+    """
+    cap = f"{SCRATCH}/contention-{tag}.txt"
+    open(cap, "w").close()
+    subprocess.run(["tmux", "-S", sock, "kill-session", "-t", f"={SESSION}"],
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)